lib/utils/location.js

```diff
@@ -45,7 +45,7 @@ export function matchesPageLocation(includes, excludes) {
 /* eslint-disable key-spacing */
 export const regexes = {
 	frontpage:        /^\/(?:hot|new|rising|controversial|top)?(?:\/|$)/i,
-	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/|$)/i,
+	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/([a-z0-9]+)\/\w*)?(?:\/|$)/i,
```

Comment:
Ah, I meant, I should definitely remove the `\/` and not just the `\/w+`.  Tests will be fixed up too!